package.json

```diff
@@ -53,6 +53,7 @@
   },
   "dependencies": {
     "formsy-react": "^0.18.1",
+    "prop-types": "^15.5.10",
     "lodash": "^4.14.1"
   },
   "peerDependencies": {
```

Comment:
I think this needs to change? And can you test by running storybook with react 16? Will require updating devDeps